store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9642,4 +9642,13 @@ TODO: delete them permanently from here
 <attr id="5003" name="zimbraCustomAppBanner" type="string" max="256" cardinality="single" optionalIn="account,cos,domain" flags="accountInfo,accountCosDomainInherited,domainAdminModifiable" since="8.8.6">
   <desc>Logo app banner URL for web client</desc>
 </attr>
+
+<attr id="5004" name="zimbraSmtpStartTlsTrustedHosts" type="string" cardinality="single" optionalIn="globalConfig,server,domain" flags="serverInherited" since="8.8.6">
+  <globalConfigValue>*</globalConfigValue>
+  <desc>Configures trusted hosts sending mail from mailboxd to MTA with starttls.
+        This parameter doesn't affect on smtp sessions to zimbraDataSourceSmtpHost.
+        *   - trust all hosts
+        host... - A whitespace separated list of trusted hosts
```

Comment:
Did you test to see the output of _zmprov desc -a zimbraSmtpStartTlsTrustedHosts_?  As in the multiline comments in the ZAttr*.java, I suspect that the line breaks in the description field will not be displayed as you expected. Why don't you describe it in the different style (other than the list style)?

Reply:
Yes, initially, I used braces ( )  and dots for better presentation with zmprov desc -a, but reviewers prefers consistency with the other attribute description in zimbra-attr.xml.

Reply:
With zmprov desc -a, we can get better readability with following description.

Description... 
(  * - trust all hosts. host[ host...] - A whitespace separated list of trusted hosts )

I'll investigate if I can use list formatter in <desc> element.

Reply:
Updated the description not to use list.